google/cloud/bigtable/examples/bigtable_grpc_credentials.cc

```diff
@@ -109,41 +99,6 @@ void JWTAccessToken(std::vector<std::string> const& argv) {
   (argv.at(0), argv.at(1), argv.at(2));
 }
 
-void GCECredentials(std::vector<std::string> const& argv) {
-  if (argv.size() != 2) {
-    throw Usage{"test-gce-credentials: <project-id> <instance-id>"};
-  }
-  // Create a namespace alias to make the code easier to read.
-  namespace cbt = ::google::cloud::bigtable;
-  namespace cbta = ::google::cloud::bigtable_admin;
-  using ::google::cloud::GrpcCredentialOption;
-  using ::google::cloud::Options;
-  using ::google::cloud::StatusOr;
-
-  //! [test gce credentials]
```

Comment:
Evidently these are used in a `.dox` file.

Reply:
Oops, fixed.